extensions/integration-tests/src/objectExplorer.test.ts

```diff
@@ -8,27 +8,31 @@
 import 'mocha';
 import * as azdata from 'azdata';
 import { context } from './testContext';
-import { getDefaultTestingServer, getBdcServer, TestServerProfile } from './testConfig';
+import { getBdcServer, TestServerProfile, getAzureServer, getStandaloneServer } from './testConfig';
 import { connectToServer } from './utils';
 import assert = require('assert');
 
 if (context.RunTest) {
 	suite('Object Explorer integration suite', () => {
 		test('BDC instance node label test', async function () {
-			assert(process.env.BDC_BACKEND_HOSTNAME !== undefined &&
-				process.env.BDC_BACKEND_USERNAME !== undefined &&
-				process.env.BDC_BACKEND_PWD !== undefined, 'BDC_BACKEND_HOSTNAME, BDC_BACKEND_USERNAME, BDC_BACKEND_PWD must be set using ./scripts/setbackenvariables.sh or .\\scripts\\setbackendvaraibles.bat');
 			const expectedNodeLabel = ['Databases', 'Security', 'Server Objects', 'Data Services'];
 			let server = await getBdcServer();
 			await VerifyOeNode(server, 6000, expectedNodeLabel);
 		});
 		test('Standard alone instance node label test', async function () {
-			const expectedNodeLabel = ['Databases', 'Security', 'Server Objects'];
-			let server = await getDefaultTestingServer();
+			if (process.platform === 'win32') {
+				const expectedNodeLabel = ['Databases', 'Security', 'Server Objects'];
+				let server = await getStandaloneServer();
+				await VerifyOeNode(server, 3000, expectedNodeLabel);
+			}
+		});
+		test('Azure SQL DB instance node label test', async function () {
+			const expectedNodeLabel = ['Databases', 'Security'];
+			let server = await getAzureServer();
 			await VerifyOeNode(server, 3000, expectedNodeLabel);
 		});
 		test('context menu test', async function () {
-			let server = await getDefaultTestingServer();
+			let server = await getAzureServer();
```

Comment:
You only test this against an azure server? What about all the other server types?